leakcanary-android-core/src/main/java/leakcanary/internal/InternalLeakCanary.kt

```diff
@@ -247,5 +254,23 @@ internal object InternalLeakCanary : (Application) -> Unit, OnObjectRetainedList
     ) as T
   }
 
+  internal enum class FormFactor {
+    MOBILE,
+    TV,
+    WATCH,
+    INSTANT_APP,
```

Comment:
instant app is not a form factor. What will we do when instant apps run on watch or TV?

Reply:
I also questioned putting `INSTANT_APP` here. Even though it's mutually exclusive today doesn't guarantee it in the future.
Will separate it out